justfile

```diff
@@ -112,12 +112,22 @@ fix: devenv
 # Run the dev project
 run:
     #!/usr/bin/env bash
-    port=$(echo $SERVER_HOST | awk -F: '{print $3}' | tr -d / )
+    port=$(echo $RELEASE_HOST | awk -F: '{print $3}' | tr -d / )
     $BIN/uvicorn hatch.app:app --reload --port $port
 
 
 token WORKSPACE="workspace":
-    $BIN/python dev-token.py "{{ WORKSPACE }}"
+    @$BIN/python dev-token.py "{{ WORKSPACE }}"
+
+
+get-index WORKSPACE="workspace":
+    @curl -s http://localhost:8001/workspace/{{ WORKSPACE }}/current -H "Authorization: $({{ just_executable() }} token {{ WORKSPACE }})"
```

Comment:
Is hardcoding the host/post deliberate?

Reply:
No good catch, but I've re-written the same functionality in python now, so it fixed via a different route